src/app/components/common/footer/footer.component.html

```diff
@@ -0,0 +1,165 @@
+          <div class="col-12">
+            <h6 class="mb-4" i18n>Stores</h6>
+            <ul class="nav">
+              <li *ngFor="let store of storeList" class="nav-item col-12">
+                <a [routerLink]="['/stores/', store.id]" class="nav-link">
+                  {{ store.name }}</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="col-12 col-md-12 col-lg-4">
+        <div class="row g-4">
+          <div class="col-12">
+            <h6 class="mb-4" i18n>Get to know us</h6>
+            <ul class="nav">
+              <li
+                *ngFor="let knowUsItem of getToKnowUsList$ | async"
+                class="nav-item col-12"
+              >
+                <a class="nav-link">{{ knowUsItem }}</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="border-top py-4">
+      <div class="row align-items-center">
+        <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+          <ul class="list-inline mb-0">
+            <li class="list-inline-item text-dark" i18n>Payment Partners</li>
+            <li class="list-inline-item">
+              <a href="#"
+                ><img src="../assets/images/payment/amazonpay.svg" alt="amazon pay"
+              /></a>
+            </li>
+            <li class="list-inline-item">
+              <a href="#"
+                ><img
+                  src="../assets/images/payment/american-express.svg"
+                  alt="american express"
+              /></a>
+            </li>
+            <li class="list-inline-item">
+              <a href="#"
+                ><img src="../assets/images/payment/mastercard.svg" alt="mastercard"
+              /></a>
+            </li>
+            <li class="list-inline-item">
+              <a href="#"
+                ><img src="../assets/images/payment/paypal.svg" alt="paypal"
+              /></a>
+            </li>
+            <li class="list-inline-item">
+              <a href="#"
+                ><img src="../assets/images/payment/visa.svg" alt="visa"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <div class="col-lg-7 mt-4 mt-md-0">
+          <ul class="list-inline mb-0 text-lg-end text-center">
+            <li class="list-inline-item mb-2 mb-md-0 text-dark" i18n>
+              Get deliveries with FreshCart
+            </li>
+            <li class="list-inline-item ms-4">
+              <a href="#">
+                <img
+                  src="../assets/images/appbutton/appstore-btn.svg"
+                  alt="appstore"
+                  style="width: 140px"
+              /></a>
+            </li>
+            <li class="list-inline-item">
+              <a href="#">
+                <img
+                  src="../assets/images/appbutton/googleplay-btn.svg"
+                  alt="google play"
+                  style="width: 140px"
```

Comment:
Do not use inline styles. Use bootstrap class or add styles in scss files or for width img has parameter "width" where you can set this